src/monodroid/jni/android-system.cc

```diff
@@ -1,8 +1,8 @@
-#include <climits>
```

Comment:
Why are we moving *away* from the C++ headers?

Are we now aiming for a [C++ Runtime Library](https://developer.android.com/ndk/guides/cpp-support) of `none`?

> There is also the option to have no STL. There are no linking or licensing requirements in that case. No C++ standard headers are available.

I'm not even sure how we'd set that in `CMakeLists.txt`.

Reply:
`<climits>` is just a wrapper around `limits.h` which is already included by other headers in the sources, it's just not needed.

Reply:
As for the STL we use `system` (in https://github.com/grendello/xamarin-android/blob/runtime-profile/build-tools/scripts/cmake-common.props#L5) which is fine for us.